src/ContactForm.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react';
+
+export default class ContactForm extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      name: '',
+      avatar: '',
+      occupation: '',
+      showContactForm: false
+    };
+  }
+
+  handleNameChange(event) {
+    this.setState({
+      name: event.target.value
+    });
+  }
+
+  handleOccupationChange(event) {
+    this.setState({
+      occupation: event.target.value
+    });
+  }
+
+  handleAvatarChange(event) {
+    this.setState({
+      avatar: event.target.value
+    });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+
+    const { name, avatar, occupation } = this.state;
+    this.props.onSubmit({ name, avatar, occupation });
+    const bool = this.state.showContactForm !== true;
+    this.setState({
+      showContactForm: bool
+    });
+  }
+
+  showHideContactForm() {
+
+    const bool = this.state.showContactForm !== true;
```

Comment:
I wouldn't use those words as variable names. You could also omit this statement and just write: 
```
this.setState({
showContactForm: !this.state.showContactForm
});
```